### PR TITLE
ENG-1176 - Show practiceThresholdMinutes on level solutions area

### PR DESCRIPTION
**Pull request by adamkecskes**
CodeCombat_Teacher_Dashboard_and_CodeCombat_-_Coding_games_to_learn_Python_and_JavaScript___CodeCombat

## Summary by CodeRabbit

- **New Features**
	- Introduced a practice threshold feature for game levels, enhancing level management.
	- Added a utility function to convert seconds into a user-friendly minutes and seconds format.
	- Expanded the teacher dashboard with new metrics related to student practice thresholds, including formatted practice threshold display.

- **Bug Fixes**
	- Improved the display of playtime in the teacher dashboard for better readability.

- **Tests**
	- Added tests for the new utility function to ensure accurate time formatting.

### CODE REVIEW

**Review by nwinter — COMMENTED** [review]
Looks good! Need `utils.secondsToMinutesAndSeconds` to apply to `practiceThresholdMinutes` display also.

File: ozaria/site/components/teacher-dashboard/Panel/components/ConceptCheckInfo.vue [hunk 1]
```diff
@@ -36,8 +41,15 @@ export default {
     >
       <img src="/images/ozaria/teachers/dashboard/svg_icons/Icon_TimeSpent.svg">
       <p>
-        <b>{{ 'Time Spent: ' }}</b>
-        {{ `${conceptCheck.timeSpent} min` }}
+        <b>{{ $t('teacher.time_played_label') }}</b>
+        {{ `${conceptCheck.timeSpent}` }}
```
Comment (smallst) [inline ozaria/site/components/teacher-dashboard/Panel/components/ConceptCheckInfo.vue:45]: if we add unit `min` in threshold, we should add unit here too. also, maybe we can add some green/red color for timeSpent which shorter/longer than the threshold for easy checking

**Review by smallst — CHANGES_REQUESTED** [review]
Others LGTM. while i think unit should be fixed

**Comment by adamkecskes** [comment]
@nwinter @smallst @mrfinch  I'm a bit confused here. Not sure how to display. 
I think, the format like `1:23` already suggests the unit, and I'm not sure how to display better? `1:23s`? or `1:23 (m:ss)?
Also, the thresholdMinutes is already minutes. Should we display `1:00` instead of `1 min`?

What should be the format then?
1) Time spent `1:12s` Practice threshold: `1:00s`
2) Time spent `1:12` Practice threshold: `1:00`
Or some other maybe?

**Comment by mrfinch** [comment]
> @nwinter @smallst @mrfinch I'm a bit confused here. Not sure how to display. I think, the format like `1:23` already suggests the unit, and I'm not sure how to display better? `1:23s`? or `1:23 (m:ss)? Also, the thresholdMinutes is already minutes. Should we display `1:00`instead of`1 min`?
> 
> What should be the format then?
> 
> 1. Time spent `1:12s` Practice threshold: `1:00s`
> 2. Time spent `1:12` Practice threshold: `1:00`
>    Or some other maybe?

We can use the 2. format you suggested everywhere for this to maintain uniformity

**Review by mrfinch — APPROVED** [review]

**Pull request merged**